keel/Core/storage_backends.py

```diff
@@ -1,15 +1,15 @@
-from django.conf import settings
+from config.settings import production
```

Comment:
well..in production also all the variables will come under settings only....
because we are just appending these files to base.py
so eventually they become part of settings only

this change can be removed